src/headers.ts

```diff
@@ -21,6 +21,8 @@ import extensionPointRegistry from "@/extensionPoints/registry";
 import "@/blocks";
 import "@/contrib";
 
+// Avoid silent failures
+process.on("uncaughtException", console.error);
```

Comment:
I'm not entirely sure why Node is hiding errors in this file, as we've seen while debugging https://github.com/pixiebrix/pixiebrix-extension/issues/800

This line makes them visible. This might be more appropriate though:
```suggestion
process.on("uncaughtException", (error) => {
  console.error(error);
  process.exitCode = 1; // Logging is shown asynchronously so we can't call .exit(1)
});
```

Reply:
> This line makes them visible. This might be more appropriate though:

@fregante I agree that changing the exit code is the right way to go. You can put that in this PR, or open a separate one

Reply:
I'm gonna test this and then commit it to this PR

Reply:
I found out why `uncaughtException` weren't shown: Rollbar was adding a handler, which by default replaces Node’s, as documented in https://nodejs.org/api/process.html#process_event_uncaughtexception

Since this PR already excludes `rollbar`, errors are already shown without this handler 🎉 so I dropped the line altogether